script.js

```diff
@@ -0,0 +1,35 @@
+
+
+$(document).ready(function(){
```

Comment:
Remember from class, this is technically only needed if you are loading your script tag earlier in your html. Since u put it right before the `</body>` you don’t actually need it